spec/system/page_objects/pages/admin_color_palette_config_area.rb

```diff
@@ -46,19 +46,23 @@ def color_palette_editor
       end
 
       def duplicate_button
-        page.find(".duplicate-palette")
+        find(".duplicate-palette")
       end
 
-      def copy_to_clipboard_button
-        page.find(".copy-to-clipboard")
+      def copy_to_clipboard
+        find(".copy-to-clipboard").click
+
+        expect(PageObjects::Components::Toasts.new).to have_success(
```

Comment:
ensuring we have actually copied before letting user do anything else with this clipboard